instance/management/commands/instance_statistics_csv.py

```diff
@@ -0,0 +1,304 @@
+        log_line.error = log_line
+
+        playbook_extra_script_arguments = '--skip-hit-statistics' if settings.INSTANCE_LOGS_SERVER_HOST else ''
+
+        # Launch the collect_activity playbook, which places a file into the `playbook_output_dir`
+        # on this host.
+        ansible.capture_playbook_output(
+            requirements_path=os.path.join(os.path.dirname(playbook_path), 'requirements.txt'),
+            inventory_str=inventory,
+            vars_str=(
+                'local_output_dir: {output_dir}\n'
+                'local_output_filename: user_statistics\n'
+                'remote_output_filename: /tmp/activity_report\n'
+                'config_section: {config_section}\n'
+                'extra_script_arguments: {extra_script_arguments}'
+            ).format(
+                output_dir=playbook_output_dir,
+                config_section=name_prefix,
+                extra_script_arguments=playbook_extra_script_arguments
+            ),
+            playbook_path=playbook_path,
+            username=settings.INSTANCE_LOGS_SERVER_SSH_USERNAME,
+            logger_=log_line,
+        )
+
+    def collect_instance_statistics(
+            self,
+            out,
+            domain_name,
+            start_date,
+            end_date
+    ):  # pylint: disable=too-many-locals
+        """Generate the instance statistics CSV."""
+        instance = self.get_instance_from_domain_name(domain_name)
+        appserver = instance.get_active_appservers().first()
+        name_prefix = appserver.server_name_prefix
+
+        self.stderr.write(self.style.SUCCESS('Running playbook...'))
+
+        with ansible.create_temp_dir() as playbook_output_dir:
+            self.get_elasticsearch_hits_data_summary(playbook_output_dir, name_prefix, start_date, end_date)
+            self.get_instance_usage_data(playbook_output_dir, name_prefix, appserver.server.public_ip)
+
+            csv_writer = csv.writer(out, quoting=csv.QUOTE_NONNUMERIC)
+            csv_writer.writerow([
+                'Fully Qualified Domain Name',
+                'Server Name Prefix',
+                'Name',
+                'Contact Email',
+                'Unique Hits',
+                'Total Hits',
+                'Total Courses',
+                'Total Users',
+                'Age (Days)'
+            ])
+
+            filenames = [os.path.join(playbook_output_dir, f) for f in os.listdir(playbook_output_dir)]
+            data = ConfigParser()
+            data.read(filenames)
+
+            try:
+                section = data[name_prefix]
+            except KeyError:
+                # Set the section to an empty dict
+                section = {}
+
+            instance_age = datetime.now(instance.created.tzinfo) - instance.created
+
+            try:
+                email = instance.betatestapplication_set.get().user.email
+            except BetaTestApplication.DoesNotExist:
+                email = 'N/A'
+
+            csv_writer.writerow([
```

Comment:
@pcockwell, when the `INSTANCE_LOGS_SERVER_HOST` is not configured (explicitly set to `''` to avoid using the Filebeat server), the existing collect_activity script collects the (unique) hits from the logs of the latest appserver (can see in the playbook output and also the output file generated on the appserver), but the output of the management command mentions `'N/A` for both fields related to the hits in its output. Can you check this on stage and fix it?